tools/cmake/CheckPythonPackage.cmake

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# CMake functions for checking for Python packages
+# Requires PYTHON_EXECUTABLE to be defined.  Call FindPython first!
+
+# set OUTPUT_VAR to whether PACKAGENAME was found
+function(check_python_package PACKAGENAME OUTPUT_VAR)
+
+    # can't have Python packages without Python!
+    if(NOT Python3_FOUND)
+        set(${OUTPUT_VAR} FALSE PARENT_SCOPE)
+        return()
+    endif()
+
+    set(NEED_TO_RUN_CHECK TRUE)
+
+    if(DEFINED ${OUTPUT_VAR})
+        if(${OUTPUT_VAR})
+
+            # if the python interpreter changed, we need to recheck
+            if("${PY_INTERP_FOR_${OUTPUT_VAR}}" STREQUAL "${Python3_EXECUTABLE}")
+                set(NEED_TO_RUN_CHECK FALSE)
+            endif()
+
```

Comment:
@multiplemonomials, Same comment as before, could you please remove the new line?